### PR TITLE
Improved menu, fixed problem with scroll

### PR DESCRIPTION
**Pull request by adudekyeti**
With custom image:
image

Without custom image:
image

### CODE REVIEW

File: modules/Users/models/Module.php [hunk 1]
```diff
@@ -122,13 +124,29 @@ public function saveLogoutHistory()
...
+	 * Get user login history.
+	 *
+	 * @param int|bool $limit
+	 *
+	 * @return array
+	 */
+	public static function getLoginHistory($limit = false)
+	{
+		return (new \App\Db\Query())->from('vtiger_loginhistory')
+			->where(['user_name' => \App\Session::get('user_name')])
```
Comment (mariuszkrzaczkowski) [inline modules/Users/models/Module.php:152]: ?

File: layouts/basic/modules/Users/Modals/LoginHistoryModal.tpl [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{*<!-- {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} -->*}
+{strip}
+	<!-- tpl-Users-PasswordModal -->
+	<div class="modal-body pb-3">
+		<div class="listViewContentDiv listViewPageDiv" id="listViewContents">
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Users/Modals/LoginHistoryModal.tpl:5]: ?

File: modules/Users/views/LoginHistory.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Login history modal view class.
+ *
```
Comment (mariuszkrzaczkowski) [inline modules/Users/views/LoginHistory.php:5]: ?

File: modules/Users/views/LoginHistory.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+	/**
+	 * Change user password.
+	 *
+	 * @param \App\Request $request
+	 */
+	public function process(App\Request $request)
+	{
+		$moduleName = $request->getModule();
+		$viewer = $this->getViewer($request);
+		$viewer->assign('MODULE_NAME', $moduleName);
```
Comment (mariuszkrzaczkowski) [inline modules/Users/views/LoginHistory.php:59]: ?

**Review by mariuszkrzaczkowski — APPROVED** [review]

File: modules/Users/views/LoginHistory.php (hunk 3 above)
Comment (adudekyeti) [reply modules/Users/views/LoginHistory.php:5]: package

File: layouts/basic/modules/Users/Modals/LoginHistoryModal.tpl (hunk 2 above)
Comment (adudekyeti) [reply layouts/basic/modules/Users/Modals/LoginHistoryModal.tpl:5]: remove class

**Pull request merged**